Jenkinsfile

```
@@ -0,0 +1,52 @@
+#!groovy
+
+pipeline {
+  agent {
+    label 'docker-runner'
+  }
+  stages{
+    stage('Run docker container') {
+      environment {
+        AWS_ACCESS_KEY_ID = credentials('AWS_ACCESS_KEY_ID')
+        AWS_SECRET_ACCESS_KEY = credentials('AWS_SECRET_ACCESS_KEY')
+      }
+      steps {
+        echo "Running irx-r-base container"
+        sh """
+        \$(aws ecr get-login --no-include-email --region us-west-2 &> /dev/null)
+        docker run -d --name ${BUILD_TAG} 579831337053.dkr.ecr.us-west-2.amazonaws.com/irx-r-base:5
+        """
+      }
+
+    }
+    stage('Build & test PKPDsim') {
+      steps {
+        echo 'Installing and checking irxtools'
```

this label is incorrect, should read 
```suggestion
        echo 'Installing and checking PKPDsim'
```